app/lib/client/lib/userRoles.js

```diff
@@ -35,6 +35,10 @@ Meteor.startup(function() {
 					}
 				} else if (role.type === 'changed') {
 					ChatMessage.update({ roles: role._id }, { $inc: { rerender: 1 } }, { multi: true });
+				} else if (role.type === 'kicked') {
```

Comment:
Since the RoomRoles is the only update happening here, you can combine both if statements into one
```
else if (role.type === 'kicked' && role.scope)
```

Reply:
Fixed! :smile: